python-sdk/tests/sql/operators/test_load_file.py

```diff
@@ -165,3 +165,42 @@ def test_tables_creation_if_they_dont_exist(database_table_fixture, if_exists):
 
     database_df = db.export_table_to_pandas_dataframe(test_table)
     assert database_df.shape == (3, 9)
+
+
+# TODO: Remove this test in astro-sdk 1.5.0
```

Comment:
For this PR scope, we are adding a warning and in 1.5.0 we will deprecate it. But we will remove it later maybe in 2.0.0 so I think this test will require even after 1.5.0 WDYT?

Reply:
As it would be deprecated in 1.5.0. So any version above that would not require this warning or test. Isn't it?

Reply:
Deprecation means they can still pass it and use it, they won't be able to pass if we will remove it.